src/style.css

```diff
@@ -1 +1,73 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir-Book.ttf) format('opentype') tech(color-COLRv1);
```

Comment:
The `tech(color-COLRv1)` part of the `src` property is not standard CSS. If you intended to use a specific technology or format, ensure it's supported by browsers or remove it.